docs/troubleshooting.md

```diff
@@ -0,0 +1,115 @@
+## No data is being sent to Datadog
+
+Use this guide when the SDK is installed, the app compiles but no data is received by Datadog.
+It is best to follow these steps in order.
+
+### Check the configuration
+
+Sometimes it can be a small mistake in the configuration. Common things to check:
+
+-   Check that your clientToken and applicationId are correct
+-   Check you have not set sessionSamplingRate to something else than 100 (100 is default), or else your session may not be sent
+-   If you’ve setup a `Proxy` in the Datadog configuration, check that it is correctly configured
+-   Check that you are tracking views (all events must be attached to a view) and sending events
+
+### Look at SDK logs in React Native
+
+-   Set `config.verbosity = SdkVerbosity.DEBUG` (import `SdkVerbosity` from `'@datadog/mobile-react-native'`)
+-   Logs will appear in the javascript console, you should expect to see an output like this one:
+
+```
+ INFO  DATADOG: Datadog SDK was initialized
+ INFO  DATADOG: Datadog SDK is tracking interactions
+ INFO  DATADOG: Datadog SDK is tracking XHR resources
+ INFO  DATADOG: Datadog SDK is tracking errors
+ DEBUG  DATADOG: Starting RUM View “Products” #Products-oaZlP_FVwGM5vtPoup_rT
+ DEBUG  DATADOG: Adding RUM Action “RCTView” (TAP)
+```
+
+In this example, the first 4 logs indicate that the SDK has been correctly configured, the last 2 ones that some events are supposed to be sent.
+
+#### Possible cause of issue
+
+If on iOS you see some DEBUG logs indicating that logs or RUM events were sent **before** the initialization logs, it might be the cause for the SDK not sending events.
+
+Currently the SDK does not support sending events before initialization, and attempting to do so will put the SDK in a state where it cannot send any data.
+
+#### Solution
+
+##### With `DdSdkReactNative.initialize`
+
+If you use `DdSdkReactNative.initialize` to start the Datadog SDK, call this function in your top-level `index.js` file so the SDK is initialized before your other events are sent.
+
+##### With `DatadogProvider`
+
+Starting from SDK version `1.2.0-beta1`, you can initialize the SDK using the `DatadogProvider` component, which includes a RUM events buffer to prevent this very issue from happening.
```

Comment:
> which includes a RUM events buffer to prevent this very issue from happening.
Perhaps refactor this part a bit in order to explicitly say what the buffering would fix in case people jump text.